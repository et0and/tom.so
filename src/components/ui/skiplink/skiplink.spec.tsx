import React from "react";
import { render } from "@testing-library/react";
import { SkipLink } from "./skiplink";

describe("<SkipLink/>", () => {
  it("renders correctly", () => {
    const { container } = render(<SkipLink />);
    expect(container).toMatchSnapshot();
  });
});
