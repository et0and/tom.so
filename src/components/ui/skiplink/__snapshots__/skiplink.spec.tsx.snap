// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SkipLink/> renders correctly 1`] = `
<div>
  <a
    class="absolute px-4 py-2 underline transition duration-75 transform -translate-x-1/2 -translate-y-full bg-white dark:bg-inherit border border-t-0 border-black sr-only left-1/2 focus:translate-y-0 focus:px-4 focus:py-2 focus:not-sr-only"
    href="#main"
  >
    Skip to content
  </a>
</div>
`;
